src/iowrap.hpp

```diff
@@ -0,0 +1,139 @@
+#ifndef STROBEALIGN_IOWRAP_HPP
+#define STROBEALIGN_IOWRAP_HPP
+
+#include <cstdint>
+#include <string>
+
+#include <zlib.h>
+#include <vector>
+#include "isa-l/igzip_lib.h"
+
+#include <thread>
```

Comment:
```suggestion
#include <thread>
#include <vector>
#include <zlib.h>
#include <isa-l/igzip_lib.h>
```
(Yes, I realize I suggested to write `#include "isa-l/..."` originally, but this is probably more consistent.)